worlds/aquaria/__init__.py

```diff
@@ -176,10 +176,17 @@ def set_rules(self) -> None:
         """
         Launched when the Multiworld generator is ready to generate rules
         """
-
+        if self.options.early_energy_form == 1:
+            self.multiworld.early_items[self.player][ItemNames.ENERGY_FORM] = 1
+        elif self.options.early_energy_form == 2:
+            self.multiworld.local_early_items[self.player][ItemNames.ENERGY_FORM] = 1
+        if self.options.early_bind_song == 1:
+            self.multiworld.early_items[self.player][ItemNames.BIND_SONG] = 1
+        elif self.options.early_bind_song == 2:
+            self.multiworld.local_early_items[self.player][ItemNames.BIND_SONG] = 1
```

Comment:
option comparisons are best done by importing the class and comparing to the class attribute,
ex. `if self.options.early_energy_form == EarlyEnergyForm.option_early:`
or by using the option string comparison (my favorite)
ex. `if self.options.early_energy_form == "early":`

Reply:
I fully agree with this suggestion. Magic number comparisons are not the best